core/config/basicConfig.js

```diff
@@ -0,0 +1,29 @@
+import harden from '@agoric/harden';
+
+import { makeBasicMintController } from './basicMintController';
+import { makeNatAssay } from './assays';
+
+const emptyConfig = harden({
+  makeCustomPayment(_issuer, _payment) {
```

Comment:
Yeah, I found this name confusing as well. The name `makeCustomPayment` suggests that it is returning the customized payment object, but it is just returning the customization methods that should be used to make the customized concrete payment object. `add` is suggests they are being imperatively added to an object rather than being used as an ingredient in a new object.

The merge of your two names, `makeCustomPaymentMethods`, is precisely accurate. But whatever we choose here, we should use it wherever we use this inheritance pattern. As such, `makeCustomFooMethods` is too long.

I suggest we merge before finding the right name. I'd rather see this merged with an accurate too-long name than an inaccurate name. Though let's remember to look for a better name afterwards.

(I'll never live down `getOwnPropertyDescriptor` ;) )

Reply:
Actually, since we're passing these methods the core-abstract-super-wrapped-original method anyway, let's keep the name `makeCustomPayment` but live up to it. The basic one would be

```js
makeCustomPayment(_issuer, superPayment) {
  return superPayment;
}
```

Instead of

```js
        makeCustomPayment(issuer, payment) {
          const delegatedUsePaymentMethods = harden({
            getUse() {
```

The `makeCustomPayment` in pixelConfig would begin

```js
        makeCustomPayment(issuer, superPayment) {
          return harden({
            ...superPayment,
            getUse() {
```

This might seem to have a comparative downside of giving the config complete control of what's in a customized purse. But the current code does that as well. We must rely on `makeCustomPayment` giving us a payment that obeys the payment spec if we gave it a payment that obeys the payment spec. We should just make clear that we are giving it that much control.

Reply:
That change is done